core/src/main/java/io/questdb/std/str/Utf8StringSink.java

```diff
@@ -102,7 +102,7 @@ public Utf8StringSink put(@Nullable Utf8Sequence us) {
 
     @Override
     public Utf8StringSink put(byte b) {
-        ascii = false;
```

Comment:
The (implicit) convention is that we never call this method on ASCII-only sequences. Maybe we should document it somehow.